docs/editor.md

```diff
@@ -104,3 +106,63 @@ it will only show the (+) button to add new cells when it has less than `maxChil
 
 It currently just controls whether the button is shown, but its still possible to add new cells by dragging.
 it will be revisited in the future and is therefore considered experimental.
+
+## Additional info
+
+Some examples of the internal JSON representation for understanding:
+
+### Simple example
+
+<p>
+  <figure align="center">
+    <img alt="Text editing plugin" src="../docs-images/json-example-1.png"><br>
+  </figure>
+</p>
+
+```json
+{
+  "id": "obknih",
+  "version": 1,
+  "rows": [
+    {
+      "id": "b27eia",
+      "cells": [
+        {
+          "id": "e9htzt",
+          "size": 12,
+          "plugin": {
+            "id": "ory/editor/core/content/slate",
+            "version": 1
+          },
+          "dataI18n": {
+            "default": {
+              "slate": [
+                {
+                  "type": "HEADINGS/HEADING-TWO",
+                  "children": [
+                    {
+                      "text": "This is a heading"
+                    }
+                  ]
+                },
+                {
+                  "type": "PARAGRAPH/PARAGRAPH",
+                  "children": [
+                    {
+                      "text": "This is some paragraph text"
+                    }
+                  ]
+                }
+              ]
+            }
+          },
+          "rows": [
+            
+          ],
+          "inline": null
+        }
+      ]
+    }
+  ]
+}
+```
```

Comment:
i think this is a nice addition. Typescript users might also want to look into the `Value` type.